packages/api/package.json

```diff
@@ -45,7 +45,8 @@
     "lint": "yarn prettier --write && yarn eslint --fix",
     "api": "node build/start api",
     "lsp_api": "node build/start lsp_api",
-    "datastore_api": "node build/start datastore_api"
+    "datastore_api": "node build/start datastore_api",
+    "ingestor_app": "node build/start ingestor_app"
```

Comment:
it might make more sense to emphasize this just runs once and is not a long running process. maybe like serverless_ingestor or serverless_write, what do you think?